tests/include/detray/test/device/cuda/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+# Detray library, part of the ACTS project (R&D line)
+#
+# (c) 2024 CERN for the benefit of the ACTS project
+#
+# Mozilla Public License Version 2.0
+
+# C++17 support for CUDA requires CMake 3.18.
+cmake_minimum_required(VERSION 3.18)
+
+# Enable CUDA as a language.
+enable_language(CUDA)
+
+# Set the CUDA build flags.
+include(detray-compiler-options-cuda)
+
+include_directories( SYSTEM $<TARGET_PROPERTY:covfie::core,INTERFACE_INCLUDE_DIRECTORIES> )
```

Comment:
Why? :confused: Why didn't you simply add `covfie::core` to the list of libraries linked to `detray_test_cuda`? You can also add it as a "private library dependency".

Generally, modifying include paths at the directory level, is a bad code smell / sign. :thinking:

Reply:
Will that also get rid of compiler warnings in the CI? Anyway, I am not even using covfie here, so removed